src/components/Home/HomeComponent.js

```diff
@@ -23,28 +44,111 @@ type Props = {
  */
 
 export default class HomeComponent extends Component<Props> {
+  constructor(props) {
+    super(props);
+    this.styleOver = this.styleOver.bind(this);
+    this.styleNotOver = this.styleNotOver.bind(this);
+    this.skipVideo = this.skipVideo.bind(this);
+  }
+
+  state = {
+    displayVideo: true
+  }
+
+  componentWillMount(): void {
+    this.styleNotOver();
+  }
+
+  componentDidMount() {
+    // subscribe state change
+    // eslint-disable-next-line
+    this.refs.player.subscribeToStateChange(this.handleStateChange.bind(this));
+  }
+
+  /**
+   * hide the video container when video's over
+   * @param state
+   */
+  handleStateChange(state) {
+    if (state.ended){
+      this.skipVideo()
+    }
+  }
+
   /**
    *  Function to create the music links
-   * @param {Array<string>} musicStyles - The array of music styles
+   * @param {Array} musicStylesLogo - The array of music styles logos
    * @returns {Array<any>}
    */
-  renderMusicStyleLinks = (musicStyles: Array<string>): Array<any> =>
-    musicStyles.map((musicStyle, index) => (
-      <li key={musicStyle}>
-        <Link to={`/${musicStyle}`}>{formattedMusicStyles[index]}</Link>
-      </li>
-    ));
+  renderMusicStyleLinks = (musicStylesLogo: Array<any>): Array<any> =>
+    musicStylesLogo.map((musicStyleLogo) => (
+    <li key={musicStyleLogo.title}>
+      <Link
+        to={`/${musicStyleLogo.title}`}
+        onMouseEnter={() => (this.styleOver(musicStyleLogo.title))}
+        onMouseLeave={this.styleNotOver}
+      >
+      <img src={musicStyleLogo.logo} alt={`${musicStyleLogo.title} style logo`}/>
+      </Link>
+    </li>
+  ));
+
+  /**
+   * function called 'onMouseEnter' on music style Link
+   * change h1 content
+   */
+  styleOver(title){
+    this.setState({
+      homeTitle: pointFreeUpperCase(title)
+    })
+  }
+
+  /**
+   * function called 'onMouseEnter' on music style Link
+   * change h1 content
+   */
+  styleNotOver(){
```

Comment:
Tu peux transformer ta fonction comme ça : 
`
styleNotOver = () => {
    this.setState({
      homeTitle: "Select your style"
    })
  }
`